administration/src/project-configs/bayern/dataPrivacyBase.tsx

```diff
@@ -449,11 +449,16 @@ export const DataPrivacyBaseText = () => (
       Informationen, die Sie unverschlüsselt per E-Mail an uns senden können, möglicherweise auf dem Übertragungsweg von
       Dritten gelesen werden.{' '}
     </p>
-    <p style={{ marginTop: '0pt', marginBottom: '8pt' }}>
+    <div>
```

Comment:
Should we readd the margin bottom here?

Reply:
hm its very inconsistent in the template. the sections directly above do not use the margin after the headline. so it looks weird imo